echo-notifications/src/main/resources/templates/swabbie/body.ftl

```diff
@@ -0,0 +1,101 @@
+<html>
+<head>
+</head>
+<body
+  style="-moz-box-sizing: border-box; -ms-text-size-adjust: 100%; -webkit-box-sizing: border-box; -webkit-text-size-adjust: 100%; Margin: 0; background-color: #FAFAFA !important; box-sizing: border-box; color: #0a0a0a; font-family: Helvetica, Arial, sans-serif; font-size: 16px; font-weight: normal; line-height: 1.3; min-width: 100%; padding: 0; text-align: left; width: 100% !important;">
```

Comment:
I would suggest skipping the `-moz-` and `-webkit-` prefixed properties. box-sizing is well supported at this point, as is hyphens, which is used below.